perfrepo-cli

```diff
@@ -347,6 +354,55 @@ class TestExecCLI(GenericCLI):
         self._perf_api.testExecution_update(testexec)
         return 0
 
+    def _do_search(self, argv):
+        i = 1
+
+        search_criteria = perfrepo.PerfRepoTestExecutionSearch()
+
+        # TODO: group, order-by, limit-from, how-many,
+        #       executed-before, executed-after
+
+        while i < len(argv):
+            if argv[i] == "ids":
+                ids = argv[i+1].split(',')
+                search_criteria.set_ids(ids)
+                i += 2
+            elif argv[i] == "uid":
+                search_criteria.set_testUid(argv[i+1])
+                i += 2
+            elif argv[i] == "name":
```

Comment:
Since this refers to a Test name (and the "uid" to Test uid) could we change the name of the parameter to something like tname/tuid? TestExecutions also have a 'name' attribute (though it's not a search criterion).